.eslintrc.js

```diff
@@ -184,7 +184,7 @@ module.exports = {
     ecmaVersion: 2020,
     sourceType: 'script',
   },
-  ignorePatterns: ['dist/**/*', 'types/**/*', 'dev/**/*'],
+  ignorePatterns: ['dist/**/*', 'types/**/*', 'dev/**/*', '!dev/update-authors.js'],
```

Comment:
Sorry if this is late to the party but should we maybe migrate this file to ts?

Reply:
> Sorry if this is late to the party but should we maybe migrate this file to ts?

I think it's feasible but would be overengineered, we need typescript for definition check only

Reply:
And update-authors scripts should be not touched by sequelize user in their app, so

Reply:
I think we don't need it to migrated to ts

Reply:
Is this file the .eslintrc file? Or the authors script?